src/main/groovy/wooga/gradle/unity/utils/internal/ProjectSettings.groovy

```diff
@@ -19,11 +19,33 @@
 package wooga.gradle.unity.utils.internal
 
 import groovy.transform.InheritConstructors
+import wooga.gradle.unity.APICompatibilityLevel
+import wooga.gradle.unity.BuildTargetGroup
+import wooga.gradle.unity.utils.GenericUnityAsset
 
 @InheritConstructors
 class ProjectSettings extends UnityAssetFile {
 
     boolean getPlayModeTestRunnerEnabled() {
         content['playModeTestRunnerEnabled'] && content['playModeTestRunnerEnabled'] == 1
     }
+
+    Map getAPICompatibilityLevelPerPlatform() {
```

Comment:
I wonder if we actually would prefer a `Map<BuildTargetGroup, ApiCompatibilityLevel>` here. Since we have problems with `BuildTargetGroup` being not very portable to different Unity versions we could use `Map<String, ApiCompatibilityLevel>` that would be the most honest version.

Reply:
I think for now using strings as the keys will be fine.